package.json

```diff
@@ -26,6 +26,12 @@
     "components",
     "component library"
   ],
+  "files": [
+    "dist",
+    "CONTRIBUTING.MD",
```

Comment:
kept this in in case an engineer digs into the `node_modules` - though may be unnecessary